files/zh-cn/mdn/community/roles_teams/index.md

```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
-  - **Yari content Web API**：负责审查 `/en-US/docs/Web/API` 下的文档
-  - **Yari content HTTP**：负责审查 `/en-US/docs/Web/HTTP` 下的文档
-  - **Yari content CSS**：负责审查 `/en-US/docs/Web/CSS` 下的文档
-  - **Yari content accessibility**：负责审查 `/en-US/docs/Web/Accessibility` 下的文档
-  - **Yari content SVG**：负责审查 `/en-US/docs/Web/SVG` 下的文档
-  - **Yari content Web Assembly**：负责审查与 WebAssembly 相关的文档
+无论在这个项目中担任什么样的角色，你总是一名[贡献者](#贡献者)。**贡献者**是最基本的角色，所有其他角色都建立在其基础之上。因此，在以任何身份参与本项目时，都必须满足贡献者角色的要求。
 
-  以下团队分别对应于不同的本地化团队：
+### 贡献者
 
-  - **Yari content ES**：负责审查 MDN 西班牙语文档
-  - **Yari content FR**：负责审查 MDN 法语文档
-  - **Yari content JA**：负责审查 MDN 日语文档
-  - **Yari content KO**：负责审查 MDN 韩语文档
-  - **Yari content PT-BR**：负责审查 MDN 巴西葡萄牙语文档
-  - **Yari content RU**：负责审查 MDN 俄语文档
-  - **Yari content ZH**：负责审查 MDN 中文文档
+贡献者，或*社区参与者*，用他们的时间、技能、意见和想法为项目添砖加瓦。他们直接参与项目工作，并为项目增添价值。除了撰写并测试代码外，创建和更新文档、调查研究、修复缺陷和帮助其他社区成员都是贡献的一种。
 
-### 加入团队意味着什么
+根据贡献频率不同，你可以是偶尔贡献的人，也可以是积极贡献的人。如果你对项目产生了重大影响，可能会被提名为[杰出贡献者](#杰出贡献者)或晋升为[组织成员](#组织成员)。
 
-我们使用 [GitHub 团队](https://docs.github.com/zh/organizations/organizing-members-into-teams/about-teams)功能来管理团队。这意味着，当你被加入到团队中时，你已经传达了更密切地参与项目的意图。这也意味着你有一些额外的责任和权利：
+如果你是新人，并且想成为一名贡献者，可以看看我们的[贡献者指南](https://github.com/mdn/content/blob/main/CONTRIBUTING.md)和 [MDN GitHub 组织中的仓库列表](https://github.com/orgs/mdn/repositories)。
 
-1. 团队中的成员通常会被添加到上述相关领域的 [CODEOWNERS](https://github.com/mdn/content/blob/main/.github/CODEOWNERS) 文件中（注：对于本地化团队，则[在此处](https://github.com/mdn/translated-content/blob/main/.github/CODEOWNERS)）。
-2. 基于 CODEOWNERS 文件，当拉取请求（PR）涉及到你职责范围内的文件时，GitHub [负载均衡算法](https://docs.github.com/zh/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team#routing-algorithms) 会自动将拉取请求分配给你。
-3. 团队成员具有更高的仓库访问权限。仅为成员分配需要的仓库的访问权限。
+作为一名贡献者，你可以通过参与以下活动来参与该项目：
 
-### 如何加入团队
+- 在[沟通渠道](/zh-CN/docs/MDN/Community/Communication_channels)中参与社区讨论。
+- 帮助其他贡献者处理拉取请求和问题，或指导新贡献者。
+- 提交缺陷报告。查看 MDN [`content`](https://github.com/mdn/content) 仓库中[你可以提出议题的类型](https://github.com/mdn/content/issues/new/choose)。如果你发现了平台的缺陷，你可以在 MDN 的 `yari` 仓库中[提出议题](https://github.com/mdn/yari/issues/new/choose)。
+- 对问题发表评论，推动对话取得丰硕成果。
+- 通过提交[拉取请求](/zh-CN/docs/MDN/Community/Pull_requests)来解决开启的议题（例如，在 [`content`](https://github.com/mdn/content/issues) 仓库中）。
+- 出席社区活动。
+- 帮助推荐 MDN 项目。
 
-要成为团队成员，你需要：
+**需求：**
 
-- 同意遵守我们的[社区参与准则](https://www.mozilla.org/zh-CN/about/governance/policies/participation/)。
-- 同意 Mozilla 的[提交访问权限的要求](https://www.mozilla.org/zh-CN/about/governance/policies/commit/requirements/)（Commit Access Requirement）。
-- 在你的 GitHub 账户上配置[双重身份验证](https://docs.github.com/zh/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication)。
+如果想要成为一名贡献者，你需要遵循：
 
-如果你想成为团队的一员，你应首先在[这个仓库提交一个议题](https://github.com/mdn/mdn/issues/new/choose)。
+- [Mozilla 行为准则](https://github.com/mdn/mdn-community/blob/main/CODE_OF_CONDUCT.md)
+- [MDN 的贡献准则](https://github.com/mdn/mdn-community/blob/main/CONTRIBUTING.md)
```

Comment:
This link is broken, and it seems that the file was never created in the mdn/mdn-community repository. Hey @bsmth. Is there any plan to creating this, or should we state that this file is located in the root directory of each repo.

https://github.com/mdn/content/blob/27e8dec205645aa69d3171e7004191a6066debf3/files/en-us/mdn/community/roles_teams/index.md?plain=1#L48-L51

Reply:
>Is there any plan to creating this, or should we state that this file is located in the root directory of each repo.

There's no plan to create this, no: we're already linking to code of conduct (Community Participation Guidelines) so contribution guide should be per project `CONTRIBUTING.md`. I'm fine with saying 'check the `CONTRIBUTING.md` file in each repo for guidance' as you suggested 👍🏻

Reply:
@jasonren0403 could you also fix this ;)

Reply:
> @jasonren0403 could you also fix this ;)

Should we point to `translated-content` in localized content, and `content` in en-US content? 🤔 

```suggestion
- [MDN 的贡献准则](https://github.com/mdn/translated-content/blob/main/CONTRIBUTING.md)
```

Reply:
Maybe something like this:

```md
- Contribution guidelines (check the `CONTRIBUTING.md` file in each repo)
```